server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -61,9 +61,9 @@ services:
       - frontera_cms_postgres_data:/var/lib/postgresql/data/portal
     container_name: frontera_cms_postgres
     environment:
-      - POSTGRES_PASSWORD=dev
-      - POSTGRES_USER=dev
-      - POSTGRES_DB=dev
+      - POSTGRES_PASSWORD=taccforever
+      - POSTGRES_USER=postgresadmin
+      - POSTGRES_DB=taccsite
```

Comment:
I recommend we change the examples to not use the words tacc in them either and make them generic to any portal (and also properly identify the role being assigned here). This should only effect local development setups running it first time out of the box with defaults.

      POSTGRES_PASSWORD=corecmsadminpassword
      POSTGRES_USER=corecmsadmin
      POSTGRES_DB=core_cms

Reply:
I am all about this rename, but not right now. Because, due dilligence requires:
1. an edit across two repositories
    - `Frontera-Portal` (two files) (this PR, two files)
    - `cms-site-template` (two files) (the relevant PR's have been merged)
2. a re-test of three scenarios
    - `Frontera-Portal` (published CMS image)
    - `cms-site-template` (stand-alone CMS image)
    - `Frontera-Portal` & `cms-site-template` (local CMS image)
3. two days before sprint end, this scope of change and testing

Reply:
I will even help test or perform or plan for this rename, in 2021. I think we should have:
- a JIRA ticket for Frontera Portal that has three sub-tasks:
    - Portal: change two files, test stand-alone, update README's (and `/docs`, as necessary)
    - CMS: change two files, test stand-alone, update README's (and `/docs`, as necessary)
    - Portal+CMS(+Docs): test local images, update Conf. docs

Reply:
Agree, the abstracting of all named things can wait until January.

Reply:
[FP-808: Anonymize Frontera Portal Setting Values](https://jira.tacc.utexas.edu/browse/FP-808)